README.md

```diff
@@ -1,2 +1,75 @@
+
+| Vocabulary | Definition |
+|---|---|
+| Bootstrap | This repo, team, and process. |
+| Ignition | The decision to move forward with the new governance |
+| Contributor | You, and anyone that interacts with this repo. |
+| TLCs | The Top Level Councils of the project – Currently the CPC and C3 (defined below) |
+| CPC | A new proposed Top Level Committee – The "Cross Project Council |
+| C3 | A new proposed Top Level Committee – The "Cross Community Council |
+| CNCF | The Cloud Native Computing Foundation (Prior Art) |
+| COC | Code of Conduct |
+| MVF | Minimum Viable Foundation. The bare minimum governance documentation to operate a function joint foundation |
+| RFC | Request For Comments. The feedback and iteration process followed in the `/programs` directory. |
+
+## Deliverables
+The deliverables from this body will be:
+
+  - [ ] A recommendations document to help inform the new Foundation's by-laws (Goal: November 1st)
+  - [ ] A recommendations document to help inform the TLCs' charters (Goal: ???)
+  - [ ] A collection of fully drafted governance documents required for Ignition. "Minimum Viable Foundation" (MVF). (Goal: March 1st)
+  - [ ] A collection of program, initiative, and governance recommendation documents not required for MVF. After Ignition these will be passed to the appropriate bodies (CPC, C3, etc.) to execute.
+
+## Operations
+
+Bootstrap follows a [Consensus Seeking][] decision making model.
+
+> Outstanding Questions:
+> If full consensus can not be achieved for a particular PR, what is the fallback voting structure?
+> If full consensus can not be achieved for Ignition, what is the fallback? 2/3rds supermajority of CommComm, TSC and TAC? Something else?
+
+The Code of Conduct explains the *bare minimum* behavior
+expectations the Node.js Foundation requires from all contributors.
+[Please read it before participating.][COC]
+
+To begin contributing to working groups and initiatives, start participating!
+
+The [`/programs`](./programs) directory contains instructions and templates for a standard RFC process for new Foundation programs proposals. To propose a new program that the joint Foundation may want to administer, please follow the directions in the `/programs` folder's [README](./programs/README.md).
+
+The [`/governance`](./governance) directory is meant as a staging ground for more well fleshed out governance documents that will closely – if not exactly – represent governance documentation that will be use to guide day to day operation of the TLCs, working groups, and projects in the Foundation. All documents in here should be the execution of accepted RFCs from `/programs`. This group will prioritize fully drafting program RFCs that are marked as required for MVF.
+
+### Committee Co-Chairs
+The committee will be co-chaired by representatives from the TSC, CommComm and TAC. At the time of writing these are:
+
+- Joe Sepi (CommComm)
+- Michael Dawson (TSC)
+- ??? (TAC)
```

Comment:
Yeah, see note written directly below it 😉I'm more than open to alternatives. Vote among committee members for their rep seems appropriate, although I'll point out that the chair responsibilities are strictly administrative (run meetings, administer agenda, etc) and should be seen as a burden, not a privilege. Chosen because Dawson is the TSC chair, and Joe offered to take up the mantle on the CommComm side since I'm looking to delegate a few things.

Want to draft language you think would work better?

Reply:
@jasnell I'm thinking I should open an issue we can discuss in next TSC meeting how to select the chair?  Make sense to you?

Reply:
The chair cannot be selected only by the node.js TSC if this committee is to represent projects in both foundations.

Reply:
@jasnell the suggestion was to have 3, one selected by each of the CommComm, TSC and TAC.  However, I think the discussion has also moved on beyond that.